src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -32,14 +34,27 @@ public void setupMyCommands(
             OperatorInterface operatorInterface,
             SetRobotHeadingCommand resetHeading,
             ArcadeDriveCommand arcade,
+            TurretRotationCommand turret,
```

Comment:
this command name is too vague. I know it's not part of your change, but can you rename it to whatever it actually does? (e.g. `TurretAutomaticRotationCommand` or `TurretControlViaJoystickCommand` or whatever represents what it's doing